lib/runtimeSettings.js

```diff
@@ -323,9 +323,17 @@ module.exports = {
     runtimeState: {
         enabled: true,
         ui: true
+    },
+    httpAdminMiddleware: function(req,res,next) {
+        res.set("Content-Security-Policy", "frame-ancestors 'self' *.flowforge.cloud *.other.flowfuse:* *.other.flowforge:*");
```

Comment:
Just to note so it doesn't get overlooked, these domains must be configurable - ideally using our existing settings that customers will have configured in their self-hosted environments.